app/src/main/java/com/example/una/TimeFormatHelper.java

```diff
@@ -0,0 +1,23 @@
+package com.example.una;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class TimeFormatHelper {
+    private static final String dayMonthYearPattern = "MM/dd/yyyy";
+    private static final String dayMonthYearHourMinutePattern = "MM/dd/yyyy HH:mm:ss";
+    private static final DateFormat dayMonthYearFormat = new SimpleDateFormat(dayMonthYearPattern);
+    private static final DateFormat dayMonthYearHourMinuteFormat = new SimpleDateFormat(dayMonthYearHourMinutePattern);
+
+    public static String getDateStringFromDate(Date date) {
+        String currentTimes = dayMonthYearFormat.format(date);
+        return currentTimes;
+    }
+
+    public static String getDateTimeStringFromDate(Date date) {
+        String currentTimes = dayMonthYearHourMinuteFormat.format(date);
+        return currentTimes;
```

Comment:
We should also add a "time since" method so that we can have the relative time on the broadcasts like we did for instagram 👍